pylint/checkers/newstyle.py

```diff
@@ -43,17 +35,6 @@
               'Used when the super builtin didn\'t receive an \
                argument.',
               {'maxversion': (3, 0)}),
```

Comment:
This also python2 only check. Why not removing this?

Reply:
As mentioned in one of my previous comments, we'd like to add a Python 2 mode so that we can still lint Python 2 files although the running interpreter is Python 3. From what I saw in my last review, we're now removing some checks that should already be handled as minimum level of compatibility.